owlbot.py

```diff
@@ -16,4 +16,11 @@
 
 import synthtool.languages.java as java
 
-java.common_templates()
+java.common_templates(
+    java.common_templates(
+        excludes=[
+            ".kokoro/build.sh",
+            ".github/sync-repo-settings.yaml",
```

Comment:
The bot warns about 4 files.

> Warning: This pull request is touching the following templated files:
> 
> .github/sync-repo-settings.yaml
> .kokoro/nightly/samples.cfg
> .kokoro/presubmit/samples.cfg
> samples/snapshot/pom.xml

(If the list is not comprehensive, then we just will get a change later)

Reply:
Uh, I should have excluded two more files in there.